ush/forecast_predet.sh

```diff
@@ -153,6 +154,32 @@ FV3_predet(){
   fi
   FV3_OUTPUT_FH="${FV3_OUTPUT_FH} $(seq -s ' ' "${fhr}" "${FHOUT}" "${FHMAX}")"
 
+  # Create an FV3 fhr list to be used in the namelist
+  # The FV3 fhr list for the namelist and the FV3 fhr list for the filenames
+  # are only different when REPLAY_ICS is set to YES
+  if [[ "${REPLAY_ICS:-NO}" == "YES"  ]]; then
+    FV3_OUTPUT_FH_NML="$(echo "scale=5; ${OFFSET_START_HOUR}+(${DELTIM}/3600)" | bc -l)"
```

Comment:
Is there a reason to call this `FV3_OUTPUT_FH_NML`?  Why can't this just be `FV3_OUTPUT_FH`?

Reply:
When `REPLAY_ICS` is set to `YES`, the fhr used in the namelist (`FV3_OUTPUT_FH_NML`) will be different from the fhr list in used in `FV3_postdet` (`FV3_OUTPUT_FH`). Th fhr list used in the namelist will have some decimal numbers (e.g. 3.08333, 6, 9, 12) and the fhr list used in FV3_postdet will have all whole numbers (e.g. 3, 6, 9, 12), which are converted to `FH3` and `FH2` then used in the filenames.